teuthology/task/install/rpm.py

```diff
@@ -132,12 +144,12 @@ def _update_package_list_and_install(ctx, remote, rpm, config):
     repos_only = config.get('repos_only')
 
     if repos:
-        log.debug("Adding repos: %s" % repos)
```

Comment:
This was a useful marker before installing repos.
Maybe it is more convenient to add wiping code before informing user we are trying to add repos?